index.markdown

```diff
@@ -817,9 +845,13 @@ If you are an intern and have more info let me know.
 	W2015.
 
 <section id="Area-SF" />
-### San Francisco
+### San Francisco, CA
 * Since SF is a city, prices are pretty high.
-* It's very tough to find housing in the city, especially for cheap.
+* It's very tough to find housing in the city, especially for cheap. No, seriously.
+  The average rent in San Francisco is
```

Comment:
Average is a misleading metric, can you just link to the article but not state the average?